llm_on_ray/inference/models/neural-chat-7b-v3-1.yaml

```diff
@@ -13,13 +13,4 @@ ipex:
 model_description:
   model_id_or_path: Intel/neural-chat-7b-v3-1
   tokenizer_name_or_path: Intel/neural-chat-7b-v3-1
-  chat_processor: ChatModelGptJ
-  prompt:
-    intro: '### System:
-      You are a chatbot developed by Intel. Please answer all questions to the best of your ability.'
-    human_id: '
-
-      ### User'
-    bot_id: '
-
-      ### Assistant'
+  chat_template: "'### System:You are a chatbot developed by Intel. Please answer all questions to the best of your ability.\n'{% if messages[0]['role'] == 'system' %}{% set loop_messages = messages[1:]%}{% set system_message = messages[0]['content'] %}{% else %}{% set loop_messages = messages %}{% set system_message = false %}{% endif %}{% for message in messages %}{% if (message['role'] == 'user') != (loop.index0 % 2 == 0) %}{{ raise_exception('Conversation roles must alternate user/assistant/user/assistant/...') }}{% endif %}{% if message['role'] == 'user' %}{{ '### User: ' + message['content'].strip() }}{% elif message['role'] == 'assistant' %}{{ '### Assistant:'  + message['content'].strip() }}{% endif %}{% endfor %}{% if add_generation_prompt %}{{'### Assistant:\n'}}{% endif %}"
```

Comment:
Should this `### System: You are ...` be included in the messages instead of in the template? The template should convert the system message to the correct format.

Reply:
Can you please add tests for any chat template that you introduced to ensure they are correct?